File: experiments/020-PythonVsJava/cells/python-pytest_1-Empty_free/2019-2c-parcial2-masked/20260921-210220-97414/analysis/own-tests-project/src/surveycrawler/turn_clockwise_command.py
from surveycrawler.command import Command


class TurnClockwiseCommand(Command):

    # character

    @classmethod
    def character(cls):
        return 'h'

    # executing

    def execute_on(self, a_crawler):
        a_crawler.turn_clockwise()

    def undo_on(self, a_crawler):
        a_crawler.turn_counter_clockwise_ignoring_ground()

File: experiments/020-PythonVsJava/cells/java-gradle_1-Empty_free/2019-2c-parcial2-masked/20260921-214358-38088/output/project/src/main/java/surveycrawler/TurnClockwiseCommand.java
package surveycrawler;

public class TurnClockwiseCommand extends RepeatableCrawlerCommand {

    // testing

    @Override
    public boolean isFor(char aCommand) {
        return aCommand == 'h';
    }

    // executing

    @Override
    protected void executeOnce(SurveyCrawler aCrawler) {
        aCrawler.turnClockwise();
    }
}
